local-cli/runWindows/utils/WindowsStoreAppUtils.ps1

```diff
@@ -150,9 +150,12 @@ function Install-AppFromAppx {
 function Start-Locally {
     param(
         [Parameter(Mandatory=$true, Position=0, ValueFromPipelineByPropertyName=$true)]
-        [string] $ID <# package.appxmanifest//Identity@name #>
-    )
+        [string] $ID, <# package.appxmanifest//Identity@name #>
 
+        [Parameter(Mandatory=$true, Position=1, ValueFromPipelineByPropertyName=$true)]
+        [string] $proxy
```

Comment:
Let's just thread the args through so this is a more general purpose utility (and make optional).